browser/module.go

```diff
@@ -74,6 +74,27 @@ func (mi *ModuleInstance) Exports() k6modules.Exports {
 
 type mapping map[string]any
 
+// wildcards is a list of extra mappings for our API (api/).
+func wildcards() map[string]string {
+	return map[string]string{
+		"Page.query":             "$",
+		"Page.queryAll":          "$$",
+		"Frame.query":            "$",
+		"Frame.queryAll":         "$$",
+		"ElementHandle.query":    "$",
+		"ElementHandle.queryAll": "$$",
+	}
+}
+
+// isWildcard returns true if the method is a wildcard method and
+// returns the name of the method to be called instead of the original
+// method.
+func isWildcard(exceptions map[string]string, typ, method string) (string, bool) {
```

Comment:
Instead of passing in the `wildcards`, can it not just call `wildcards` from `isWildCard`? A simpler approach could be to define the `wildscards` as a global in the package.

```go
var wildcards = map[string]string{
  "Page.query":             "$",
  "Page.queryAll":          "$$",
  "Frame.query":            "$",
  "Frame.queryAll":         "$$",
  "ElementHandle.query":    "$",
  "ElementHandle.queryAll": "$$",
}

func isWildcard(typ, method string) (string, bool) {
	name := typ + "." + method
	s, ok := wildcards[name]
	return s, ok
}
```

Reply:
That makes sense to keep it simple. However, we'd need to add a `nolint` directive (which we don't want #58). Also, the function lets us make the `wildcards` map read-only.